WcaOnRails/app/webpacker/lib/requests/routes.js.erb

```diff
@@ -1,3 +1,9 @@
+function jsonToQueryString(json) {
+  return Object.keys(json)
+    .filter((key) => json[key] !== undefined && json[key] !== null)
+    .map((key) => `${encodeURIComponent(key)}=${encodeURIComponent(json[key])}`).join('&');
+}
```

Comment:
See my remark about vanilla JS in another PR

Reply:
(Copying my response in the other PR)
Actually, a use-case that we want is, if the JSON is having any value as null, we shouldn't pass that value to the server. So, this code actually removes the keys which have value as null.